modules/cluster-logging-rn-5.4.3.adoc

```diff
@@ -0,0 +1,34 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-4-3"]
+= Logging 5.4.3
+This release includes link:https://access.redhat.com/errata/RHBA-2022:5556[RHBA-2022:5556-OpenShift Logging Bug Fix Release 5.4.3].
+
+[id="openshift-logging-5-4-3-bug-fixes"]
+== Bug fixes
+* Before this update, the ElasticSearch Operator did not respect the `referencePolicy` property of the `ImageStream` custom resource when selecting an `oauth-proxy` image, which caused the Kibana deployment to fail in certain environments. With this update, using `referencePolicy` resolves the issue and the Operator can deploy Kibana successfully. (link:https://issues.redhat.com/browse/LOG-2791[LOG-2791])
+
+* Before this update, alerting rules for the `ClusterLogForwarder` custom resource did not take multiple forward outputs into account. This update resolves the issue.(link:https://issues.redhat.com/browse/LOG-2640[LOG-2640])
+
+* Before this update, clusters configured to forward logs to Amazon CloudWatch wrote rejected log files to temporary storage, causing cluster instability over time. With this update, chunk backup for CloudWatch has been disabled, resolving the issue. (link:https://issues.redhat.com/browse/LOG-2768[LOG-2768])
+
```

Comment:
I think we need below bugs for 5.4.3
https://issues.redhat.com/browse/LOG-2791
https://issues.redhat.com/browse/LOG-2536

Reply:
@kabirbhartiRH LOG-2791 is there, preview doesn't update. LOG-2536 does not need RNs per @periklis because it will be part of the 5.5 release.